extensions/hibernate-validator/deployment/pom.xml

```diff
@@ -41,6 +41,16 @@
             <artifactId>assertj-core</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-resteasy-jsonb-deployment</artifactId>
```

Comment:
@stuartwdouglas I reckon you will have to rebase and change that one to runtime in your ClassLoader PR?

Reply:
thanks for the heads up.